sea/app.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, root_path, env):
         self.name = os.path.basename(root_path)
         self.env = env
         self.config = self.config_class(root_path, self.default_config)
+        # only filter default configurations
+        self.config.filter_env()
```

Comment:
move to https://github.com/shanbay/sea/blob/master/sea/__init__.py#L31
@apocelipes